### PR TITLE
Bug fix#1 b

### LINKED ISSUE #1: ActivitiesController maken

### PR DESCRIPTION
**Pull request by WouterKvG**
Closes #1

### CODE REVIEW

**Comment by WouterKvG** [comment]
@steffansluis @bartist Please review!

**Comment by WouterKvG** [comment]
I think the coverage should be brought up before this is merged, could you look into that @steffansluis?

**Comment by WouterKvG** [comment]
Controllers can only be tested when routes are implemented I think. For the remaining helper, model and serializer tests should still be added.

File: app/controllers/api_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class ApiController < ApplicationController
+  include ApiHelper
+
+  # after_action :verify_policy_scoped, except: :create
```
Comment (steffansluis) [inline app/controllers/api_controller.rb]: Is this going to be used? If not, I would remove it

File: app/models/activity.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class Activity
+  include Mongoid::Document
+  include Mongoid::Timestamps
+
+  field :engine, type: String, default: -> { self.class.engine }, pre_processed: true
+
+  validates :engine, presence: true
+
+  # :nocov:
```
Comment (steffansluis) [inline app/models/activity.rb:9]: Does this need a closing 'tag'?

File: spec/controllers/v1/activities_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    describe 'when enrolled or owner' do
+      describe 'should return the activity' do
+        before { action.call }
+
+        it { expect(JSON.parse(response.body)['activity']['id']).to eq(activity2.id.to_s) }
+      end
+    end
+  end
+
+  # describe 'POST #create' do
```
Comment (steffansluis) [inline spec/controllers/v1/activities_controller_spec.rb:58]: I'm guessing these are the tests you want to fix before merging? :smile:

File: spec/helpers/activities_helper_spec.rb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+RSpec.describe ActivitiesHelper, type: :helper do
+  describe '#ActivitiesHelper' do
+    describe '#engine?' do
+      it 'returns whether or not the activity is an engine' do
+        allow(controller).to receive(:params).and_return(controller: 'bepstore/activities')
+        expect(helper.engine?).to eq(false)
+      end
+
+      it 'returns whether or not the activity is an engine' do
+        allow(controller).to receive(:params).and_return(controller: 'bepstore/goals')
```
Comment (steffansluis) [inline spec/helpers/activities_helper_spec.rb:12]: Shouldn't tests for goals be in the corresponding engine?

Comment (WouterKvG) [inline spec/helpers/activities_helper_spec.rb:12]: This is just an example, not necessarily correlated to api-goals

**Pull request merged**